src/main/java/gim/ui/HelpWindow.java

```diff
@@ -15,9 +16,23 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://ay2223s1-cs2103t-t15-4.github.io/tp/";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2223s1-cs2103t-t15-4.github.io/tp/UserGuide.html";
+    public static final String HELP_MESSAGE = USERGUIDE_URL;
 
+    public static final String HELP_PARA = "Hi there, welcome to GIM! Here is a list of supported commands.\n"
+            + "1) :a - Adds an exercise\n"
+            + "2) :d - Deletes an exercise\n"
+            + "3) :ls - Lists all exercise entries\n"
+            + "4) :find - Finds an exercise with a particular name\n"
+            + "5) :clear - Clears the saved exercises and resets the data in the system\n"
+            + "6) :sort - Sorts the exercises by date, if they have the same date, by name \n"
+            + "7) :range - Displays the exercises saved over a range of days\n"
+            + "8) :pr - Displays the Personal Record associated with a certain exercise\n"
```

Comment:
Similar thought as the above comment. :pr allows user to display all PRs?

Reply:
Necessary changes have been made, thanks